pkg/controller/instance/apply_conventions.go

```diff
@@ -25,22 +25,20 @@ import (
 
 const basePath = "/kustomize"
 
-// metadata contains metadata associated with current PlanExecution
+// metadata contains metadata associated with current plan being executed
 type metadata struct {
```

Comment:
Also, `metadata` as a name is pretty broad. wdyt about `engineMetadata`? or maybe this one is more of `executionMetadata` and the existing one is more like `kudoResourceMetadata`?